RELEASE_NOTE.md

```diff
@@ -0,0 +1,76 @@
+# Release v0.13.0
+
+
+## New Features
+
+* PAI protocol:
+  - Introduce PAI protocol and job submission v2 ([#2260](https://github.com/microsoft/pai/pull/2260))
+  - Add new job submission v2 plugin ([#2461](https://github.com/microsoft/pai/pull/2461))
+
+* Web portal:
+  - Add new home page ([#2544](https://github.com/microsoft/pai/pull/2544), [#2614](https://github.com/microsoft/pai/pull/2614))
+  - Add new user management page ([#2726](https://github.com/microsoft/pai/pull/2726), [#2796](https://github.com/microsoft/pai/pull/2796))
+
+
+## Improvements
+
+* PAI protocol:
+  - Update example jobs in marketplace v2 for PAI protocol ([#2827](https://github.com/microsoft/pai/pull/2827))
+
+* Web portal:
+  - Refine styles in job pages ([#2829](https://github.com/microsoft/pai/pull/2829))
+  - Refine alert message in job pages ([#2698](https://github.com/microsoft/pai/pull/2698))
+  - Reduce build bundle size ([#2715](https://github.com/microsoft/pai/pull/2715))
```

Comment:
what's this feature for?

Reply:
this is an improvement, not a feature. it makes building and visiting webportal faster

Reply:
Okey, let's change it to: Reduce the build bundle size to improve webportal performance.

Reply:
updated